core/scripts/bareos-config-lib.sh.in

```diff
@@ -75,19 +72,50 @@ is_function()
 #     fi
 # }
 
+contains() {
+    haystack="$1"
+    needle="$2"
+    if [ "${haystack#*"$needle"}" != "$haystack" ];then
+        return 0    # $needle is in $haystack
+    else
+        return 1    # $needle is not in $haystack
+    fi
+}
+
 info()
 {
-    echo "Info: $@" >&2
+    echo "Info: $*" >&2
 }
 
 warn()
 {
-    echo "Warning: $@" >&2
+    echo "Warning: $*" >&2
 }
 
 error()
 {
-    echo "Error: $@" >&2
+    echo "Error: $*" >&2
+}
+
+check_parameters()
```

Comment:
bareos-config-lib.sh is not only intended to be used from database creation scripts. Also this function does not only check the parameter, it also sets the `FORCE` environment variable. You may want to rename it to something like: `handle_database_scripts_command_line_parameter` ???